com.unity.multiplayer.mlapi/Runtime/Messaging/CustomMessageManager.cs

```diff
@@ -161,7 +161,19 @@ public static void UnregisterNamedMessageHandler(string name)
         /// <param name="security">The security settings to apply to the message</param>
         public static void SendNamedMessage(string name, ulong clientId, Stream stream, string channel = null, SecuritySendFlags security = SecuritySendFlags.None)
         {
-            ulong hash = NetworkedBehaviour.HashMethodName(name);
+            ulong hash = 0;
+            switch (NetworkingManager.Singleton.NetworkConfig.RpcHashSize)
+            {
+                case HashSize.VarIntTwoBytes:
```

Comment:
If you look at ILSpy, does this `case` statement get optimized out?

Reply:
no, `NetworkingManager.Singleton.NetworkConfig.RpcHashSize` is not a compile-time constant so the check happens at runtime still.